Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:boron-alpine
+MAINTAINER Mario Behling <mb@mariobehling.de>
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+RUN apt-get update && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# copy requirements
+COPY package.json /usr/src/app/
+COPY app.json /usr/src/app/
+COPY tslint.json /usr/src/app/
```

Comment:
Same here. No `tslint.json` file is present.

Reply:
I think it is https://github.com/fossasia/loklak_search/blob/master/tslint.json